### ISSUE TITLE
Add sampling

### ISSUE THREAD

**Issue by marksverdhei** [marksverdhei/bert-bot#9]
We should try out an alternative for bert-insert,
where we sample from the softmax of the logits instead of picking the highest every time.
This can lead to some interesting results. I suggest top-k sampling to not make it too nonsense-prone either

**Comment by LBlend** [comment]
I support this!
